specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -6962,6 +6962,11 @@
         "version": {
           "type": "string",
           "description": "Specifies the version of the platform image or marketplace image used to create the virtual machine. The allowed formats are Major.Minor.Build or 'latest'. Major, Minor, and Build are decimal numbers. Specify 'latest' to use the latest version of an image available at deploy time. Even if you use 'latest', the VM image will not automatically update after deploy time even if a new version becomes available."
+        },
+        "exactVersion": {
```

Comment:
is there any user impact since this changed a stable version?

Reply:
for VMScaleSet VMs, there was only 1 field 'version' returning the decimal version before. Now this new field returns the decimal version. the existing 'version' could return either 'latest' or decimal values of the version ( depending on what the user passed. ). CRP is already following this behavior.  To be precise, we have now made the VMScaleSet VM behavior consistent with regular VM behavior.

Reply:
thanks for explanataion. So this change will not be breaking change to customers of this stable version via rest api, SDks, CLIs, right?